README.md

```diff
@@ -109,11 +109,36 @@ Otherwise :violin: kindly ask someone from the dev team to help troubleshoot.
 
 ### Optional configuration options
 
-| Environment variable          | Default   | Description
-| ----------------------------- | --------- | -----------
-| ENABLE_KAFKA                  | false     | Feature flag to send kafka messages when a collection is published
-| KAFKA_ADDR                    | localhost:9092 | Comma seperated list of kafka brokers
-| KAFKA_CONTENT_PUBLISHED_TOPIC | content-published | Kafka topic to send content-published messages to
+| Environment variable                              | Default                                     | Description
+| --------------------------------------------------| --------------------------------------------| -----------
+| DEFAULT_WEBSITE_URL                               | "http://localhost:8080"                     | Service URL
+| DEFAULT_SLACK_WARNING_CHANNEL                     | "slack-client-test"                         | Slack channel
+| DEFAULT_SLACK_ALARM_CHANNEL                       | "slack-client-test"                         | Slack alarm channel
+| DEFAULT_SLACK_USERNAME                            | "Zebedee"                                   | Slack user
+| DEFAULT_PUBLIC_WEBSITE_URL                        | "http://localhost:8080"                     | Service public URL
+| DEFAULT_FLORENCE_URL                              | "http://localhost:8081"                     | Florence URL
+| DEFAULT_BRIAN_URL                                 | "http://localhost:8083"                     | Brian URL
+| DEFAULT_TRAIN_URL                                 | "http://localhost:8084"                     | Train URL
+| DEFAULT_DYLAN_URL                                 | "http://localhost:8085"                     | Dylan URL
+| CONTENT_DIRECTORY                                 | "content"                                   | Content directory
+| AUDIT_DB_ENABLED_ENV_VAR                          | "audit_db_enabled"                          | Audit DB
+| MATHJAX_SERVICE_URL                               | "http://localhost:8888"                     | Mathjax service URL
+| DATASET_API_URL                                   | "http://localhost:22000"                    | Dataset API URL
+| IMAGE_API_URL                                     | "http://localhost:24700"                    | Image API URL
+| KAFKA_ADDR                                        | "localhost:9092"                            | The address of Kafka
```

Comment:
Two of the same Env Variables in table.  Also note one has quotes, one does not.